TWLight/applications/views.py

```diff
@@ -867,11 +868,53 @@ def get_context_data(self, **kwargs):
                 editor__user__groups__name='restricted'
                 )
         app_outputs = {}
-
+        stream_outputs = []
+        list_unavailable_streams = []
+        
         for app in apps:
             app_outputs[app] = get_output_for_application(app)
-
+            stream_outputs.append(app.specific_stream)
+        
         context['app_outputs'] = app_outputs
+        
+        # This part checks to see if there are applications from stream(s) with no accounts available.
+        # Additionally, supports send_partner template with total approved/sent applications.
+        partner = self.get_object()
+
+        total_apps_approved = Application.objects.filter(
+            partner=partner, status=Application.APPROVED).count()
+
+        total_apps_sent = Application.objects.filter(
+            partner=partner, status=Application.SENT).count()
+
+        total_apps_approved_or_sent = total_apps_approved + total_apps_sent
```

Comment:
Apologies that this has only just occurred to me, but including approved apps here duplicates what's listed on the current send screen (applications listed at /send are 'approved'). So if you have 10 accounts available, 6 sent apps, and 3 approved apps, the platform will think you're out of accounts because 9 (approved or sent) + 3 (listed) > 10.

Really you don't need to count what's listed on send, you can just total 'approved or sent' and see if that's greater than the total available.

Reply:
can't believe I missed this. Should be fixed now